package.json

```diff
@@ -1,19 +1,25 @@
 {
-  "name": "iobio-components",
-  "version": "1.0.0",
-  "description": "Overture Integration of Iobio.io Genomic Analysis Tools",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/overture-stack/iobio-components"
-  },
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/overture-stack/iobio-components/issues"
-  },
-  "homepage": "https://github.com/overture-stack/iobio-components#readme"
+	"name": "@overture-stack/iobio-components",
+	"version": "1.0.0",
+	"description": "Overture Integration of Iobio.io Genomic Analysis Tools",
+	"main": "index.js",
+	"type": "module",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"prettier": "prettier --config prettier.config.js --write ./**",
+		"bam": "cd bam.iobio && pnpm run start"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/overture-stack/iobio-components"
+	},
+	"author": "",
+	"license": "ISC",
```

Comment:
Can this be changed to GNU Affero General Public License v3.0 and add root License file? 
https://github.com/overture-stack/stage/blob/develop/LICENSE

Reply:
License file added; the package.json field is handled differently in different places, for now I've kept it as `"license": "AGPL-3.0-or-later"` as found here: https://github.com/overture-stack/arranger/blob/develop/modules/components/package.json#L30

Reply:
hold on, gents... I've got info that's missing here for decision making:

1. good call, @ciaranschutte about using the right license
2. happy to see @demariadaniel acting on that feedback so quickly. all license files/references can safely be removed from this repo, because:
3. all overture apps are covered under the license set in the `.github` repo (which applies to the whole overture-stack org)

Reply:
Removed here: https://github.com/overture-stack/iobio-components/pull/15/commits/0e0deeabb3e7c967e8585e04c45b3986866be7d9